west.yml

```diff
@@ -157,7 +157,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: 0690c03f1540e8a5082da4c2b997e64bde8f4765
+      revision: pull/278/head
```

Comment:
update the `mcuboot` to the `898a1ca64a759541d9fcc37fef921db93d99ad70` which has C6 support.